website/site/layouts/partials/footer.html

```diff
@@ -180,6 +180,23 @@
             </li>
         </ul>
     </div>
+</footer>  */}}
```

Comment:
Is there a reason this commented code is being retained?

Reply:
@sbishel Removed it.